src/local-history/local-history-manager.ts

```diff
@@ -153,6 +153,16 @@ export class LocalHistoryManager {
                 uri: historyFilePath,
                 parentFileName: fileFullPath.base
             };
+
+            const activeEditorHistories = this.historyFiles.filter(history => history.filepath === document.fileName);
```

Comment:
@Anasshahidd21 Please change `history.filepath` to `history.uri`

Reply:
Thank you @kaiyue0329 for your review, I fixed that part of the issue.